### PR TITLE
Azure Compute v2 Update 

### PR DESCRIPTION
**Pull request by ayman-m**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added three new commands

## Minimum version of Cortex XSOAR
- [x ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ x] No

## Must have
- [ ] Tests
- [x ] Documentation

### CODE REVIEW

File: Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py [hunk 1]
```diff
@@ -243,6 +243,68 @@ def create_vm_parameters(args, subscription_id):
...
+        if not private_ip_address:
+            err_msg = 'You have chosen to assign a "Static" IP address value to the interface, ' \
+                      'so you must enter a value for the "private_ip_address" argument.'
+            raise Exception(err_msg)
+        nic['properties']['ipConfigurations']['properties']['privateIPAddress'] = private_ip_address
+
+    if network_security_group:
+        network_security_group_id = f"/subscriptions/{subscription_id}/resourceGroups/"
+        network_security_group_id += f"{resource_group}/providers/Microsoft.Network/networkSecurityGroups/" \
+                                     f"{network_security_group}"
```
Comment (bziser) [inline Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py:302]: Why did you split it to 2 lines?

**Review by bziser — COMMENTED** [review]
Great work! Let's arrange a quick demo on the new commands

File: Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py (hunk 1 above)
Comment (ayman-m) [reply Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py:302]: a long line if not splited, to avoid lint warning

**Comment by ayman-m** [comment]
@bziser there was another PR merged before merging this one, I am gonna rebase the master and create a new branch with a new PR, will close this one

**Pull request closed without merging**